### PR TITLE
Add issue template

### LINKED ISSUE #2057: Add instructions to the new issue template to check for new plugin version
Consider implementing GPE-like notifications when an update is available.  Such notifications would be useful especially in light of issues like #2055.

### PR DESCRIPTION
**Pull request by briandealwis**
Put the template under `.github` to avoid cluttering the root directory.

Fixes #2057

### CODE REVIEW

**Comment by chanseokoh** [comment]
I don't see the directory `.github`. Maybe you missed it. Also, I'm guessing the template may not work when a user comes in through our _Submit Issue or Feedback..._ menu item, for which we explicitly provided some contents.

**Comment by briandealwis** [comment]
My guess is that the template will be used for people who enter issues directly via GitHub.com.

**Comment by chanseokoh** [comment]
What do you think of `BugReportCommandHandler.BODY_TEMPLATE`? (It's also for general feedback.)

File: plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java [hunk 1]
```diff
@@ -31,8 +31,20 @@
   private static final String BUG_REPORT_URL =
       "https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/new";
 
+  //@formatter:off
+  // should be kept up to date with .github/ISSUE_TEMPLATE.md
   private static final String BODY_TEMPLATE =
-      "- Cloud Tools for Eclipse Version: {0}\n- OS: {1} {2}\n- Java Version: {3}\n\n";
+      "- Cloud Tools for Eclipse Version: {0}\n"
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java]: Suggestion to upgrade to the latest CT4E? (FYI, markdown doesn't seem to work here.)

Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java]: It's a bit unfortunate we have to duplicate the template here :-(

I don't think there's any difference in appearance (i.e., markdown rendering) between launching https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/new directly or from the menu handler.

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java]: Ah, my mistake about the markdown.

BTW, you need to fix the `BugReportCommandHandlerTest` too. Also you will want to merge with master to get rid of the AERI error on Mars.

File: plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java [hunk 2]
```diff
@@ -31,8 +31,22 @@
   private static final String BUG_REPORT_URL =
       "https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/new";
 
+  //@formatter:off
+  // should be kept up to date with .github/ISSUE_TEMPLATE.md
   private static final String BODY_TEMPLATE =
-      "- Cloud Tools for Eclipse Version: {0}\n- OS: {1} {2}\n- Java Version: {3}\n\n";
+      "(please ensure you are running the latest version of CT4E with _Help > Check for Updates_)\n"
+      + "- Cloud Tools for Eclipse Version: {0}\n"
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java]: Nit: `version`? (no capitalization)

**Pull request merged**